docs/floor-filter.md

```diff
@@ -51,44 +51,28 @@ The following properties enable customization or localization of text displayed
 
 ## Usage
 
-WPF:
+### .NET MAUI:
 
 ```xml
 <Grid>
-    <esri:MapView x:Name="MyMapView" />
-    <esri:FloorFilter GeoView="{Binding ElementName=MyMapView}" />
+    <esri:MapView x:Name="mapView" />
+    <toolkit:FloorFilter GeoView="{x:Reference mapView}" />
 </Grid>
 ```
 
-MAUI:
+### UWP/WinUI:
 
 ```xml
 <Grid>
-    <Grid.RowDefinitions>
-        <RowDefinition Height="*" />
-        <RowDefinition Height="Auto" />
-    </Grid.RowDefinitions>
-    <Grid.ColumnDefinitions>
-        <ColumnDefinition Width="48" />
-        <ColumnDefinition Width="*" />
-    </Grid.ColumnDefinitions>
-    <esri:MapView
-        x:Name="MyMapView"
-        Grid.RowSpan="2"
-        Grid.ColumnSpan="2" />
-    <toolkit:FloorFilter
-        Grid.Row="1"
-        Grid.Column="0"
-        Margin="8,8,8,32"
-        GeoView="{x:Reference MyMapView}" />
+    <esri:MapView x:Name="mapView" />
+    <esri:FloorFilter GeoView="{Binding ElementName=mapView}" />
 </Grid>
 ```
 
-UWP:
+### WPF:
+
+The usage in WPF is identical to UWP/WinUI with one important distinction: `FloorFilter` is accessed with the same namespace prefix as `GeoView`.
 
 ```xml
-<Grid>
-    <esri:MapView x:Name="MyMapView" />
-    <esri:FloorFilter GeoView="{Binding ElementName=MyMapView}" />
-</Grid>
-```
\ No newline at end of file
+xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013"
```

Comment:
I liked it better when this was declared on a root-element of the XAML sample, rather than a sub-note